### PR TITLE
EC-2688 Added terms controls

### PR DESCRIPTION
**Pull request by andyjmaclean**

### CODE REVIEW

File: app/views/migration/_form.html.erb [hunk 1]
```diff
@@ -80,6 +80,8 @@
         <% end %>
         <%= agg.add_nested_fields_link(:edm_hasViews, i18n.t('buttons.object.add')) %>
       <% end %>
+      <%= story.input(:content_policy_accept, label: i18n.t('labels.story.content_policy_accept', href: '//www.europeana.eu/portal/en/rights/contributions.html').html_safe, as: :boolean, required: true,  boolean_style: :inline, label_input_html: {class: 'full-width'}, input_html: {'data-error-msg-key': 'contribute.campaigns.migration.form.validation.content-policy-accept'})%>
```
Comment (rwd) [inline app/views/migration/_form.html.erb]: The link in this should be language-agnostic, i.e. not include /en

File: app/views/migration/_form.html.erb [hunk 2]
```diff
@@ -80,6 +80,8 @@
...
+      <%= story.input(:display_and_takedown_accept, label: i18n.t('labels.story.display_and_takedown_accept', email: 'takedown@europeana.eu').html_safe, as: :boolean, required: true,  boolean_style: :inline, label_input_html: {class: 'full-width'}, input_html: {'data-error-msg-key': 'contribute.campaigns.migration.form.validation.display-and-takedown-accept'})%>
```
Comment (rwd) [inline app/views/migration/_form.html.erb]: The label for this should use "be" instead of "being".

Comment (lbiedinger) [reply app/views/migration/_form.html.erb]: updated in localeapp

File: app/views/migration/_form.html.erb [hunk 3]
```diff
@@ -80,6 +80,8 @@
         <% end %>
         <%= agg.add_nested_fields_link(:edm_hasViews, i18n.t('buttons.object.add')) %>
       <% end %>
+      <%= story.input(:content_policy_accept, label: i18n.t('labels.story.content_policy_accept', href: '//www.europeana.eu/portal/rights/contributions.html').html_safe, as: :boolean, required: true,  boolean_style: :inline, label_input_html: {class: 'full-width'}, input_html: {'data-error-msg-key': 'contribute.campaigns.migration.form.validation.content-policy-accept'})%>
```
Comment (rwd) [inline app/views/migration/_form.html.erb]: Hashes should have spaces between braces and contents, on this line and the next.

Comment (andyjmaclean) [reply app/views/migration/_form.html.erb]: I've made those changes

File: app/views/migration/_form.html.erb [hunk 4]
```diff
@@ -80,6 +80,8 @@
         <% end %>
         <%= agg.add_nested_fields_link(:edm_hasViews, i18n.t('buttons.object.add')) %>
       <% end %>
+      <%= story.input(:content_policy_accept, label: i18n.t('labels.story.content_policy_accept', href: '//www.europeana.eu/portal/rights/contributions.html').html_safe, as: :boolean, required: true,  boolean_style: :inline, label_input_html: {class: 'full-width'}, input_html: { 'data-error-msg-key': 'contribute.campaigns.migration.form.validation.content-policy-accept' })%>
+      <%= story.input(:display_and_takedown_accept, label: i18n.t('labels.story.display_and_takedown_accept', email: 'notice@europeana.eu').html_safe, as: :boolean, required: true,  boolean_style: :inline, label_input_html: {class: 'full-width'}, input_html: { 'data-error-msg-key': 'contribute.campaigns.migration.form.validation.display-and-takedown-accept' })%>
```
Comment (rwd) [inline app/views/migration/_form.html.erb]: @andyjmaclean whitespace is still needed in `label_input_html` hashes.

**Pull request merged**